lib/processHandler.js

```diff
@@ -80,37 +81,31 @@ module.exports = function HandleUsingProcess(grunt, options, commands, callback)
     cucumber.on('close', function(code) {
         if (options.isHtml) {
 
-            var featureJsonOutput,
-                featureOutput;
+            var featureJsonOutput;
+            var jsonOutput;
 
             if (options.executeParallel) {
-                featureOutput = JSON.stringify(jsonFile.readFileSync(options.output + '.json'));
+                jsonOutput = JSON.stringify(jsonFile.readFileSync(options.output + '.json'));
             } else {
                 var jsonOutputFilePath = options.output + '.json';
-                var jsonOutput;
+
 
                 if (fs.existsSync(options.output + '.json')) {
                     jsonOutput= JSON.stringify(jsonFile.readFileSync(jsonOutputFilePath));
                 } else {
-                    jsonOutput = Buffer.concat(buffer).toString();
+                    grunt.log.error('Unable to find cucumberjs json output file.');
```

Comment:
let's log more specific error message `grunt.log.error('Unable to find cucumberjs json output file \'%s\'', jsonOutputFilePath);`